### PR TITLE
[FlexTensors] define mimetype for flexible tensors

### PR DESCRIPTION
**Pull request by jaeyun-jung**
add definition to handle flexible tensors stream.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

File: gst/nnstreamer/include/tensor_typedef.h [hunk 1]
```diff
@@ -86,6 +90,14 @@
 #define GST_TENSORS_CAP_DEFAULT \
     GST_TENSORS_CAP_WITH_NUM(GST_TENSOR_NUM_TENSORS_RANGE)
 
+/**
+ * @brief Caps string for the caps template of flexible tensors.
+ * This mimetype handles non-static, flexible tensor stream without specifying the data type and shape of the tensor.
+ * The maximum number of tensors in a buffer is 16 (NNS_TENSOR_SIZE_LIMIT).
+ */
+#define GST_FLEX_TENSORS_CAP_DEFAULT \
```
Comment (gichan-jang) [inline gst/nnstreamer/include/tensor_typedef.h:98]: How about a `GST_TENSORS_FLEX_CAP_DEFAULT` for consistency?

Comment (jaeyun-jung) [reply gst/nnstreamer/include/tensor_typedef.h]: updated

**Review by myungjoo — APPROVED** [review]
LGTM. Good Start!

**Review by gichan-jang — APPROVED** [review]
LGTM

**Review by kparichay — APPROVED** [review]
LGTM

**Pull request merged**